src/main/resources/bitbucketpullrequestbuilder/bitbucketpullrequestbuilder/BitbucketBuildTrigger/help-buildChronologically.html

```diff
@@ -0,0 +1 @@
+Build Pull Requests in reverse order - older request will be build first
```

Comment:
If you change current logic to build PRs chronologically, then, I guest it would be better to rename proposed option to `reverseOrder` instead `buildChronologically`